docs/topics/practices.rst

```diff
@@ -35,12 +35,17 @@ Here's an example showing how to run a single spider with it.
         ...
 
     process = CrawlerProcess({
-        'USER_AGENT': 'Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1)'
+        'USER_AGENT': 'Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1)',
+        'FEED_FORMAT':'json',
+        'FEED_URI':'items.json'
     })
 
     process.crawl(MySpider)
     process.start() # the script will block here until the crawling is finished
 
+Define settings within dictionary in
+CrawlerProcess. FEED_FORMAT and FEED_URI are the equivalent to "-o items.json" when using the scrapy crawl shell command.
```

Comment:
> Define settings within dictionary in CrawlerProcess.

I think the example makes this clear, and the paragraph below does encourage reading the documentation of `CrawlerProcess`.

> FEED_FORMAT and FEED_URI are the equivalent to "-o items.json" when using the scrapy crawl shell command.

This information here feels out of the blue to me. If we really think that users will continue to have doubts despite the example featuring `FEED_FORMAT` and `FEED_URI`, then maybe we should add an entry to the FAQ instead, and really address the specific issue of “How to I use -o with CrawlProcess?”.

Reply:
I totally agree with you, telling that you can define any setting within a dictionary as a parameter of CrawlerProcess should be enough. There shouldn't be an extra emphasis on any specific setting. Shall I make a new pull request or can you mark it as "needs work"?